console/billing/concepts.mdx

```diff
@@ -5,16 +5,143 @@ meta:
 content:
   h1: Billing - Concepts
   paragraph: This page explains all the concepts related to Billing
-tags: billing account pricing
+tags: billing account pricing payment
 categories:
   - account
   - console
 ---
 
 
 
+
+
+<Concept opened>
+
+## Billing alerts 
+
+The Billing Alerts functionality within the Scaleway console allows users to oversee and control their expenses efficiently. This feature allows you to create alerts that notify users when predefined budget thresholds are met. Your alerts can be seamlessly delivered through SMS, email, or API webhook channels.
+
+</Concept>
+
+<Concept opened>
+
+## Billing API
+
+Scaleway's Billing API provides a quick overview of your cloud consumption and allows you to monitor expenses programmatically.
+
+</Concept>
+
+<Concept opened>
+
+## Billing contact
+
+The billing contact is an additional contact who will receive the invoice or payment alerts by email. This allows you to add a person who may not have access to the Scaleway console, but who should be aware of billing, payment documents and notifications.
+
+</Concept>
+
+<Concept opened>
+
+## Billing cycle
+
+Invoices are issued each month based on the resource consumption from the previous month. Our reference billing period is the definition of a calendar month in the Coordinated Universal Time (UTC) time zone. The automatic payment of invoices occurs in the first few days of the month, once the invoice is issued and processed using the preferred payment method of your account.
+
+</Concept>
+
+<Concept opened>
+
+## Billing information
+
+Consists of information about the organization that will be billed. They will appear on the feature and can be modified on the Scaleway console.
+
+</Concept>
+
+<Concept opened>
+
+## Consumption pie chart
+
+The consumption pie chart reflects the amount of money spent on the products you've used. Its monetary value is calculated in real-time, providing an instant overview of your expenses.
+
+</Concept>
+
+<Concept opened>
+
+## Cost Manager 
+
+Cost manager is a comprehensive and user-friendly tool to monitor and analyze cloud resource consumption, costs, and trends. It will offer a clear and detailed view of consumption data, enabling users to accurately reconcile costs for re-invoicing purposes.
+
+</Concept>
+
+<Concept opened>
+
+## Free Tier 
+
+The Free Tier offers certain products at no cost and up to a specified usage limit. These limits do not have an expiration date but may be subject to change. Eligible customers can enjoy these select services free of charge when creating an account, with a usage limit in place.
+
+</Concept>
+
+<Concept opened>
+
+## Free Trial
+
+The Free Trial provides users access to specific products at no cost. However, it is available only for a limited duration and usage quantity.
```

Comment:
```suggestion
A free trial provides users access to specific products at no cost. However, it is available only for a limited duration and usage quantity.
```

Reply:
In what circumstances do we offer a free trial?

Reply:
I think this refers to Object Storage only, for now. you can tell them it's available for OS and include a link to the doc page that talks about the free trial